app/views/taxes/_form.html.slim

```diff
@@ -0,0 +1,26 @@
+= form_with(model: [@car, tax]) do |form|
+  - if @tax.errors.any?
+    div style="color: red"
+      h2 = "#{pluralize(@tax.errors.count, "error")} prohibited this tax from being saved:"
+      ul
+        - @tax.errors.each do |error|
+          li = error.full_message
+
+  div
+    = form.label :name, style: "display: block"
+    = form.text_field :name
+
```

Comment:
amount filed is missing

Reply:
Yes, I forgot it and I am going to add it. Saw it after I committed.